mirage/scenarios/collections.ts

```diff
@@ -1,8 +1,49 @@
 import { ModelInstance, Server } from 'ember-cli-mirage';
-
+import CollectionModel from 'ember-osf-web/models/collection';
+import { CollectionSubmissionReviewStates } from 'ember-osf-web/models/collection-submission';
+import LicenseModel from 'ember-osf-web/models/license';
+import faker from 'faker';
 import { Permission } from 'ember-osf-web/models/osf-model';
 import User from 'ember-osf-web/models/user';
 
+/**
+ * ProjectModel
+ *
+ * @description A simple abstraction to simplify creation of projects in different
+ * review states
+ */
+interface ProjectModel {
```

Comment:
Minor nit, this name is a bit misleading, as this doesn't actual contain the project model.  Since it's an interface describing the shape of the expected arguments of projectBuilder, a straightforward name like `ProjectBuilderArgs` may better suit this.